### PR TITLE
 Added zinc to more groups including hardware

### LINKED ISSUE #25236: Hard to craft items fixes: Rocket Fuel Zinc availability
**Describe the bug**  
As originally reported in #25074, some recipes are unnecessarily difficult to craft.

    Rocket fuel
    Diff: 4 -> expected time 11.3 mins
    Craft time: 5 mins
    Chance of success per try: 44.2%
    Avg. tries to have one success: 2.26

Chances to learn are pretty good, the problem is key component - zinc. The only place to find it is chem room in school - both in regular and the one in necropolis. Both are occurring not too often and both usually require a lot of fighting to get in. Neither it is guaranteed that you will find any zinc in first or second school you stumble upon.
To add fuel to the flame of frustration the recipe itself is almost useless. Rocket fuel is only used in home-made rockets - two of them require 50 RF and one - 75. And from a single drop of zinc (100) you will be able to craft 20 of rocket fuel.

**Expected behavior**  
Most recipes are reasonably straightforward to craft.

**Additional context**  
Zinc should spawn in more places. The factory seems like an obvious place, since a large use of zinc in the US is for corrosion resistance of iron parts.

### PR DESCRIPTION
**Pull request by y2s82**
#### Summary
SUMMARY: Balance Increased the availability of zinc in the world.

#### Purpose of change
As per fixes #25236
Zinc is one of the ingredients of the rocket fuel.  The rocket fuel is supposedly something easy to create, and yet, its component is very difficult to come by.  This fix aims to introduce zinc in more locations.  

#### Describe the solution
The chem_zinc_powder is added to chem_home, chem_group, and hardware item groups.

#### Describe alternatives you've considered
I wondered if it was possible to also add zinc to some of the components, but that may have made them difficult to build as well.  Still, zinc is a very common compound found in paint and other household materials.  If our character can learn to separate zinc from such materials, that would make it more realistic and easier to source.

#### Additional context
Please let me know if you have more suggestions

### CODE REVIEW

**Comment by ZhilkinSerg** [comment]
You need summary line, e.g.:

```SUMMARY: Balance Increased the availability of zinc in the world.```

**Comment by mlangsdorf** [comment]
Please add zinc to the items in the factory from the More Locations mod.

**Comment by y2s82** [comment]
Thank you for the input.  I will apply the changes later today.

**Comment by y2s82** [comment]
I have taken a look at data/mods/More_Locations/factory and currently am stuck.  I have checked out the tri_tower to get some hints and realized I have to add new element to the map template.  Am I allowed to just add zinc on random location?  Does this factory get used for different context?

**Comment by y2s82** [comment]
Please let me know if this is what I am supposed to do.
I am still trying to understand how a building is put together: what does all of those acronym mean?
Any suggestion would be welcomed.

**Comment by ZhilkinSerg** [comment]
I will try to take a look tomorrow.

File: data/mods/More_Locations/factory/factory_be_gate_01.json [hunk 1]
```diff
@@ -40,6 +40,11 @@
                 "t": { "monster": "mon_turret" },
                 "Z": { "monster": "mon_secubot" },
                 "z": { "monster": "mon_skitterbot" }
+            },
+            "mapping": {
+                "H": {
+                    "item": { "item": "zinc_powder", "count": 50 },
```
Comment (ZhilkinSerg) [inline data/mods/More_Locations/factory/factory_be_gate_01.json]:
```suggestion
                    "item": { "item": "zinc_powder", "count": 50 }
```

**Pull request merged**